thriftbp/server.go

```diff
@@ -69,20 +75,20 @@ type ServerConfig struct {
 // and protocol to serve the given TProcessor which is wrapped with the
 // given ProcessorMiddlewares.
 func NewServer(cfg ServerConfig) (*thrift.TSimpleServer, error) {
-	var transport *thrift.TServerSocket
+
```

Comment:
don't start indented blocks with a blank line; this has come up a few times -- don't do it :)